torch/onnx/__init__.py

```diff
@@ -408,8 +408,8 @@ def wrapper_maker(fn):
 
         def wrapper(*args, **kwargs):
             output = fn(*args, **kwargs)
-            flat_args = tuple(function._iter_variables(args))
-            if not any(map(torch._C._jit_is_tracing, flat_args)):
+            flat_args = tuple(function._iter_None_variables(args))
+            if not any(map(lambda x: False if x is None else torch._C._jit_is_tracing(x), flat_args)):
```

Comment:
I meant here - check only first argument

Reply:
Yup, the following vars/tensors are not used in symbolic.